packages/cli/src/dbt/targets/Bigquery/index.ts

```diff
@@ -93,7 +105,13 @@ export const convertBigquerySchema = async (
         };
     }
     const lineErrorMessages = (validate.errors || [])
-        .map((err) => `Field at ${err.instancePath} ${err.message}`)
+        .map((err) => {
+            if (err.keyword === 'oneOf') {
+                return 'Profile should contain either `dataset` or `schema`';
+            }
+
+            return `Field at ${err.instancePath} ${err.message}`;
```

Comment:
Sometimes a validation error occurs without an `instancePath`. In such cases, should we simply log the error message and completely ignore the `instancePath`? I believe a similar issue arose with a protobuf breaking change, making the error confusing to interpret.

Reply:
I've pushed a commit to use `better-ajv-errors` that other targets where using instead of doing our own parsing.

Note: In that commit I added also to snowflake target since that one and bigquery were the only targets not using `better-ajv-errors`

Reply:
sweet. thank you!